PathPlanning/DynamicWindowApproach/dynamic_window_approach.py

```diff
@@ -48,9 +47,10 @@ def __init__(self):
         self.yaw_rate_resolution = 0.1 * math.pi / 180.0  # [rad/s]
         self.dt = 0.1  # [s] Time tick for motion prediction
         self.predict_time = 3.0  # [s]
-        self.to_goal_cost_gain = 0.15
+        self.to_goal_cost_gain = 0.2
         self.speed_cost_gain = 1.0
-        self.obstacle_cost_gain = 1.0
+        self.obstacle_cost_gain = 2.0
```

Comment:
Could you please keep previous parameter as default value and change parameter in the test function?

Reply:
thanks for your patient replay, the file is modified as you suggested.
this is really a tough pr, while i have learned a lot from this process, thanks again for all of your kind suggestion!